### PR TITLE
master to main

### PR DESCRIPTION
**Pull request by jiwon-lieb**
## Proposed changes

With the branch name change, some of the links in the docs had to be changed as well.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/dapp/json-rpc/api-references/admin.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ endpoint management.
 
 The `nodeInfo` administrative property can be queried for all the information known about the running
 Klaytn node at the networking granularity. These include general information about the node itself as a
-participant of the [devp2p](https://github.com/ethereum/devp2p/blob/master/README.md) P2P
+participant of the [devp2p](https://github.com/ethereum/devp2p/blob/main/README.md) P2P
```
Comment (kjhman21) [inline docs/dapp/json-rpc/api-references/admin.md:17]: this is Ethereum repo and the repo does not changed to main from master. Please roll back the change.

Comment (jiwon-lieb) [inline docs/dapp/json-rpc/api-references/admin.md:17]:
```suggestion
participant of the [devp2p](https://github.com/ethereum/devp2p/blob/master/README.md) P2P
```

Comment (jiwon-lieb) [inline docs/dapp/json-rpc/api-references/admin.md]:
```suggestion
participant of the [devp2p](https://github.com/ethereum/devp2p/blob/master/README.md) P2P
```

**Review by praveen-klaytn — APPROVED** [review]

**Comment by jiwon-lieb** [comment]
@aidan-kwon @kjhman21 이거 승인 좀 해주세요 ㅠ.ㅠ

**Review by kjhman21 — APPROVED** [review]

**Pull request merged**